OpenEdXMobile/src/test/java/org/edx/mobile/view/WebViewActivityTest.java

```diff
@@ -0,0 +1,63 @@
+package org.edx.mobile.view;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.app.ActionBar;
+import android.text.TextUtils;
+import android.view.View;
+import android.webkit.WebView;
+
+import org.edx.mobile.R;
+import org.edx.mobile.test.BaseTestCase;
+import org.edx.mobile.view.dialog.WebViewActivity;
+import org.junit.Test;
+import org.robolectric.Robolectric;
+import org.robolectric.RuntimeEnvironment;
+import org.robolectric.Shadows;
+import org.robolectric.shadows.ShadowWebView;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertNotNull;
+import static junit.framework.Assert.assertTrue;
+
+public class WebViewActivityTest extends BaseTestCase {
+
+    /**
+     * Testing method for displaying web view dialog
+     */
+    @Test
+    public void test_StartWebViewActivity_LoadsUrlAndShowsTitle() {
+        final String url = "https://www.edx.org";
+        final String title = "title";
+        test_StartWebViewActivity_LoadsUrlAndShowsTitle(url, title);
+        test_StartWebViewActivity_LoadsUrlAndShowsTitle(url, null);
```

Comment:
This should probably be converted to a parameterized test runner once we have one that works with Robolectric. Maybe add a TODO comment to that effect for this test method.